packages/desktop-ui-lib/src/lib/permission-manager/permission-manager.component.html

```
@@ -0,0 +1,455 @@
+						<div class="col-6 d-flex justify-content-end">
+							<ng-container [ngTemplateOutlet]="statusBadge"
+								[ngTemplateOutletContext]="{ status: status$ | async }">
+							</ng-container>
+						</div>
+					</div>
+
+					<!-- Action Buttons -->
+					<div class="row">
+						<div class="col-12 d-flex flex-wrap gap-2">
+							<button
+								nbButton
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)